tensorboard/plugins/graph/BUILD

```diff
@@ -156,9 +154,13 @@ py_library(
     srcs_version = "PY3",
 )
 
-tf_ng_web_test_suite(
-    name = "karma_test",
-    deps = [
-        "//tensorboard/plugins/graph/tf_graph_common:test_lib",
-    ],
-)
+# This test suite is disabled as lodash does not seem to get properly bundled
+# into the test after upgrade to spec_bundle(). Further investigation is
+# required.
+# TODO: Reenable or decide to delete the tests.
```

Comment:
These are, I think, the only tests from our old polymer code base that survived the migration to polymer3 in 2020. It's not entirely surprising that these would give me the most problems. I would be sad to end up deleting these but, well, then this part of our code base would be just as poorly covered as the rest of the polymer3 codebase.

Reply:
Yes, good idea to have a bug. Googlers see b/258801848.